custom/project_scrum_agile_extended/models/project_scrum_product_backlog.py

```diff
@@ -0,0 +1,173 @@
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import _, api, fields, models
+
+
+class ProjectScrumProductBacklog(models.Model):
+    _inherit = "project.scrum.product.backlog"
+
+    @api.depends(
+        "tasks_id",
+        "tasks_id.estimate_adjustment",
+        "tasks_id.effective_hours",
+        "tasks_id.planned_hours",
+        "tasks_id.progress",
+        "tasks_id.weightage",
```

Comment:
add tasks_id.stage_id,tasks_id.product_backlog_id.task_hours,sprint_id sprint_id.expected_hours in depends